episodes/02-image-basics.md

```diff
@@ -881,20 +881,22 @@ two orders of magnitude smaller than the bitmap version.
 ## Comparing lossless versus lossy compression (optional, not included in timing)
 
 Let us see a hands-on example of lossless versus lossy compression.
-Once again, open a terminal and navigate to the `data/` directory.
+Open a terminal (or Windows PowerShell) and navigate to the `data/` directory.
 The two output images, `ws.bmp` and `ws.jpg`, should still be in the directory,
 along with another image, `tree.jpg`.
 
 We can apply lossless compression to any file by using the `zip` command.
 Recall that the `ws.bmp` file contains 75,000,054 bytes.
 Apply lossless compression to this image by executing the following command:
-`zip ws.zip ws.bmp`.
+`zip ws.zip ws.bmp` 
+(`Compress-Archive ws.bmp ws.zip` with PowerShell).
 This command tells the computer to create a new compressed file,
 `ws.zip`, from the original bitmap image.
-Execute a similar command on the tree JPEG file: `zip tree.zip tree.jpg`.
+Execute a similar command on the tree JPEG file: `zip tree.zip tree.jpg` 
+(`Compress-Archive tree.jpg tree.zip` with PowerShell).
 
 Having created the compressed file,
-use the `ls -al` command to display the contents of the directory.
+use the `ls -al` command (`dir` with PowerShell) to display the contents of the directory.
```

Comment:
Can you use the `dir` command with options (equivalent to `-al`) in PowerShell?

Reply:
@mkcor I don't think additional options are needed as `dir` outputs in a format similar to `ls -l` by default, importantly including file size (at least for me).

Reply:
I think `dir` and `ls` are synonymous for `Get-ChildItem` in PowerShell. These return these columns:
```
Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
```
Length I believe is file size in bytes.

Reply:
Ok, good to know. And how about displaying hidden files and folders (equivalent to `-a`)?

Reply:
There's a -Force flag which I think does this but it isn't really needed here as the files we are interested in are not hidden

Reply:
@mkcor and @bobturneruk I am inclined to merge this. Please speak up in the next few days if you would like to propose further changes.

Reply:
Ok, then I would suggest removing the `-a` flag here.